docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,113 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables ({issue}`24447`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance of reading from Delta Lake tables by compressing files
+  with `ZSTD` by default, instead of the previous `SNAPPY`.  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+* Add support for views. ({issue}`24242`)
```

Comment:
Thanks for adding the docs entry for the SQL support section. Question though .. do we need to add more specific info like .. where are those views stored? Do they survive a cluster restart?

Reply:
And are create, drop and alter supported?

Reply:
The implementation is copied from the memory connector. The views are stored in memory, same as the tables are, and we only mention this for tables. There are tests that prove we support CREATE VIEW, DROP VIEW and ALTER VIEW statements.